home-tasks/src/test/java/com/epam/tc/hw1/functions/AdditionOperationTestDouble.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.tc.hw1.functions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import com.epam.tc.hw1.providers.DataProviderForCalculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class AdditionOperationTestDouble {
+    Calculator calculator;
+
+    @BeforeMethod
+    public void setUp() {
+        calculator = new Calculator();
+    }
```

Comment:
These test classes have a lot in common. At least they test calculator's operations. So would be better to make them extend some parent class, where test configuration occurs (before/after methods)